docs/axelar/intro-to-axelar.md

```diff
@@ -0,0 +1,103 @@
+# Axelar Integration
+
+The Axelar-XRPL integration enables messages to pass between the XRPL and other Axelar-supported blockchains. This integration enables the transfer of XRP and XRPL-issued tokens to other chains and back; this also works for briding non-XRPL tokens from their native chains to XRPL.
+
+In addition to cross-chain features, the Axelar Amplifier network utilizes tickets to enable multiple in-flight transactions to improve throughput.
+
+At a high-level, there are two main components that enable the Axelar-XRPL integration. On the XRPL-side, a multisig account is created; this account handles all incoming and outgoing transactions to the Axelar network by:
+
+- Holding and releasing assets as native XRP assets are moved off and on-chain.
+- Issuing and burning wrapped assets as non-native XRP assets are bridged to XRPL.
+
+The multisig account is ultimately controlled by the Axelar Network validators, which serve as signers. The XRPL multisig account signers and their weights are kept in sync with the state of the validators on the Axelar network. The Axelar validators' are responsible for verifying transactions on multiple chains, minting and burning as assets are bridged.
```

Comment:
More details here
The XRPL - Axelar connection is secured by an XRPL account owned by Axelar validators. The top 32 validators, chosen by those staking the most AXL on the Axelar PoS network, make up the signer list for the Axelar XRPL account. These 32 validators control the multi-sig account and must agree before sending transactions across the Axelar network. Interested developers can find more details on the security of the Axelar network [here](https://axelar.network/blog/security-at-axelar-core).